src/igvfd/schemas/changelogs/prediction_set.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`prediction_set.json`*
 
+### Minor changes since schema version 4
+
+* Add linking to reference files for `large_scale_gene_list` and `large_scale_loci_list`.
```

Comment:
Add ReferenceFile linkTo to large_scale_gene_list and large_scale_loci_list.

Reply:
changed